server/extension/extension.go

```diff
@@ -709,7 +720,8 @@ func (m *Manager) CallExtension() func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
-		prepareRequest(r, extName, app, m.username(r.Context()))
+		groups := m.groupsClaim(r.Context(), []string{"groups"})
```

Comment:
The list of scopes is a configurable settings in ArgoCD rbac configMap and should be retrieved from the settings object.

Reply:
done